### PR TITLE
iozone_windows: Add a case to test the stability of guest

### PR DESCRIPTION
**Pull request by YongxueHong**
Add a case to test the stability of guest by running iozone
on multiple disks in a long time.

ID: 1715710
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

File: generic/tests/iozone_windows.py [hunk 1]
```diff
@@ -50,10 +51,46 @@ def get_driver():
                 driver_name = None
         return driver_name
 
+    def run_iozone_parallel(timeout):
+        """ Run the iozone parallel. """
+        iozone_sessions = []
+        iozone_threads = []
+        thread_maps = {}
+
+        for _ in range(len(disk_letters)):
```
Comment (hereischen) [inline generic/tests/iozone_windows.py]: `for _ in disk_letters:` as disk_letters is a list

Comment (YongxueHong) [reply generic/tests/iozone_windows.py]: yes, you are right, I forgot it. thanks.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

File: generic/tests/cfg/iozone_windows.cfg [hunk 2]
```diff
@@ -12,3 +12,20 @@
             image_aio = native
         - aio_threads:
             image_aio = threads
+        - long_time_stress:
+            format_disk = yes
+            disk_index = "1 2"
+            disk_letter = "I J"
+            disk_fstype = "fat32 fat"
+            run_iozone_parllel = yes
```
Comment (lolyu) [inline generic/tests/cfg/iozone_windows.cfg]: parallel?

Comment (YongxueHong) [reply generic/tests/cfg/iozone_windows.cfg]: it's `parallel` :)

**Comment by YongxueHong** [comment]
@kimi1978wy  pls help review, thanks.

**Comment by kimi1978wy** [comment]
Acked by: Yu Wang <wyu@redhat.com>

Ran on win10-64 and win7-32, passed (changed the stress time), function completed.
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.iozone_windows.long_time_stress: PASS (695.83 s)
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.iozone_windows.long_time_stress: PASS (742.45 s)

**Comment by kimi1978wy** [comment]
Ran on win7-32, passed
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.iozone_windows.long_time_stress: PASS (43519.08 s)

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**